<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>timerConfig</class>
 <widget class="QWidget" name="timerConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>246</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Appearance</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>50</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="showTitleCheckBox">
     <property name="text">
      <string>Show title:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="KLineEdit" name="titleLineEdit"/>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="hideSecondsCheckBox">
     <property name="text">
      <string>Hide seconds</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Actions on Timeout</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="KLineEdit" name="messageLineEdit"/>
   </item>
   <item row="7" column="3">
    <widget class="KLineEdit" name="commandLineEdit"/>
   </item>
   <item row="8" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="1">
    <widget class="QCheckBox" name="showMessageCheckBox">
     <property name="text">
      <string>Show a message:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QCheckBox" name="runCommandCheckBox">
     <property name="text">
      <string>Run a command:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>showMessageCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>messageLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>runCommandCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>commandLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showTitleCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>titleLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>14</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
